packages/app-store/bigbluebutton/lib/index.ts

```diff
@@ -0,0 +1,4 @@
+export { default as VideoApiAdapter } from "./VideoApiAdapter";
+export { BbbApi } from "./bbbApi";
+export { applyMeetingUrlTemplate } from "./applyMeetingUrlTemplate";
+export { hashAttendee } from "./hashAttendee";
```

Comment:
Used separate export sources to prevent `crypto` polyfill from being included in booking page bundle.